client/src/components/contact.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+ReactModal.setAppElement('#app');
+
+function Contact(props) {
+  return (
+    <div>
+      <div id="Hide" onClick={props.handleOpenModal}>
+        Contact
+      </div>
+      <ReactModal
```

Comment:
Anything extra we should know about how you're using this library API? A comment may be good here

Reply:
Added a comment overviewing react-modal library.